pod-simulator/main.go

```diff
@@ -19,7 +19,7 @@ func main() {
 	// os.Open() opens specific file in
 	// read-only mode and this return
 	// a pointer of type os.
-	file, err := os.Open("./simulation.log")
+	file, err := os.Open("/var/logs/simulation.log")
```

Comment:
var logs is a system directory, in our context this is application data.  It should go in `/etc/pod-simulator` or be passed in as an argument.  I would opt for the argument approach, it gives us much more flexibility in the future.

Reply:
This will get addressed in https://github.com/MrSupiri/rancher-logging-explorer/issues/15